src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -924,9 +924,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         auto lock = _terminal->LockForWriting();
         _compositionScale = scale;
 
-        _renderer->TriggerFontChange(::base::saturated_cast<int>(dpi),
-                                     _desiredFont,
-                                     _actualFont);
+        _updateFont();

```

Comment:
```suggestion
        // _updateFont relies on the new _compositionScale set above
        _updateFont();
```

I had to look into the code to see this, but this is only a _nit_ :)

Reply:
Makes sense! Added